app/next.config.js

```diff
@@ -1,16 +1,21 @@
-/** @type {import('next').NextConfig} */
-
+// @ts-check
 const { i18n } = require("./next-i18next.config");
+const sassOptions = require("./scripts/sassOptions");
 
+/**
+ * Configure the base path for the app. Useful if you're deploying to a subdirectory (like GitHub Pages).
+ * If this isn't an empty string, you'll need to set the base path anywhere you use relative paths,
+ * like in `<a>`, `<img>`, or `<Image>` tags. Next.js handles this for you automatically in `<Link>` tags.
+ * @see https://nextjs.org/docs/api-reference/next.config.js/basepath
+ */
+const basePath = "";
```

Comment:
for my own understanding, should this be `"/"` instead of empty string?

Reply:
No, empty string is the default that Next.js uses. We could set it as `undefined` to make it clearer that it's using the default?

Reply:
sure, no strong feelings. could also be a comment too

Reply:
I think `undefined` makes sense, will switch it to that, and add a comment with an example